### ISSUE TITLE
Consideration for aria-hidden true on focusable element to not be considered hidden content

### ISSUE THREAD

**Issue by mraccess77** [act-rules/act-rules.github.io#1271]
This issue is being created to keep track of public discussion on whether focusable content with aria-hidden true should be considered  visible in terms of rules.

Use of aria-hidden does not impact on the browser's rendering of controls visually or to my knowledge taking something that is visible out of the focus order.  Thus, when a control has aria-hidden true and is focusable it generally made available to users of assistive technology by the browser on focus.  In this case it's necessary for the control to have proper accessible names, etc. checked by rules.

**Comment by Jym77** [comment]
Elements with `aria-hidden="true"` are **visible**, that's pretty clear (unless otherwise hidden).

Question is whether they are **in the accessibility tree** or not. They shouldn't, but when they are focusable, they might be, as you point out.

I haven't found anything about this in the specs. The closest thing I've seen is the "[Presentational role conflict resolution](https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none)" (in short: `role="none"` + focusable => ignore explicit role). While the situation you describe (`aria-hidden="true"` + focusable) is similar ("thing that should remove from accessibility tree" + focusable), it does not seem to be handled by the specs (and maybe it should…)

While running [tests on presentational role conflict resolution](https://github.com/act-rules/act-rules.github.io/wiki/Tests-on-Presentational-role-Conflict-resolution), I also tried `aria-hidden="true"`. My findings seem to indicate that Chrome is giving precedence to "focusable" (i.e. exposing elements with `aria-hidden="true"`+focusable) but Edge and Firefox are giving precedence to `aria-hidden` (i.e. not exposing those elements).

Of course, if you have more test results on situations like this, please share them 😃 

---
Our current course of action in #1189 has been to lampshade a bit the problem. Given that browsers behave inconsistently and that specs don't say who is right, it's hard to take a stance… So the best we can do is highlight the issue and tell users (of ACT rules) that this point is actually depending on there accessibility support baseline…

**Comment by JAWS-test** [comment]
See https://github.com/w3c/aria/issues/1185

**Comment by WilcoFiers** [comment]
We're not sure what to do with this issue. If aria-hidden changes, we'll adopt it in ACT when we update to a new version of ARIA.
